ci/deps/bigquery.yml

```diff
@@ -1,2 +1,3 @@
-google-cloud-bigquery>=1.12.0
-pydata-google-auth
+google-cloud-bigquery>=1.12.0,<2.0.0dev
+protobuf>=3.12.0
```

Comment:
is this not a transitive dep?

Reply:
It is `google-cloud-bigquery -> google-api-core -> protobuf`. Added to see if that prevented getting too old of a version in CI, but seems I actually needed to update the recipe here: https://github.com/conda-forge/ibis-framework-feedstock/pull/48

Will remove in my next commit.